package/doc/sphinx/source/documentation_pages/loading_files.rst

```diff
@@ -0,0 +1,156 @@
+consult the relevant documentation page.
+
+
+Generally the format of a file is automatically detected from the extension,
+for example a file called `system.xyz` is recognised as an XYZ file.
+This can be overriden by supplying the `topology_format` and `format` keyword
+arguments to Universe.
+A full list of valid values for these keywords are given in the below table.
+
+
+.. note:: It is possible to pass tarballed/zipped versions of files.  The
+	  format detection will work around this.
+
+
+
++------------------+-----------+----------+--------------+-----+
+| Source           | Extension | Topology | Trajectory   | I/O |
++==================+===========+==========+==============+=====+
+| Generic          | XYZ       | Yes      | Yes          | r/w |
+|                  +-----------+----------+--------------+-----+
+|                  | PQR       | Yes      | Yes          | r/w |
++------------------+-----------+----------+--------------+-----+
+| Protein          | MMTF      | Yes      | Yes          | r   |
+| Databank         +-----------+----------+--------------+-----+
+|                  | PDB,      | Yes      | Yes          | r/w |
+|                  | ENT,      |          |              |     |
+|                  | XPDB      |          |              |     |
++------------------+-----------+----------+--------------+-----+
+| CHARMM           | PSF       | Yes      | No           | r   |
+|                  +-----------+----------+--------------+-----+
+|                  | DCD       | Minimal  | Yes          | r/w |
+|                  +-----------+----------+--------------+-----+
+|                  | CRD       | No       | Yes          | r   |
++------------------+-----------+----------+--------------+-----+
+| Desmond          | DMS       | No       | Yes          | r   |
++------------------+-----------+----------+--------------+-----+
+| DL Poly          | CONFIG    | Yes      | Yes          | r   |
+|                  +-----------+----------+--------------+-----+
+|                  | HISTORY   | Yes      | Yes          | r   |
++------------------+-----------+----------+--------------+-----+
+| GAMESS           | GMS,      |          | Yes          | r   |
+|                  | LOG,      |          |              |     |
+|                  | OUT       |          |              |     |
++------------------+-----------+----------+--------------+-----+
+| LAMMPS           | DATA      | Yes      | Yes          | r/w |
+|                  +-----------+----------+--------------+-----+
+|                  | DCD       | Minimal  | Yes          | r/w |
++------------------+-----------+----------+--------------+-----+
+| Tinker           | TXYZ      | Yes      | Yes          | r   |
++------------------+-----------+----------+--------------+-----+
+| Tripos           | MOL2      | Yes      | Yes          | r/w |
++------------------+-----------+----------+--------------+-----+
+
+
+.. _Supported formats:
```

Comment:
We link to the coordinates and topology formats table in many places (including web site and wiki). We will need to update all of them...